### ISSUE TITLE
Error in Grow from seeds when using undo

### ISSUE THREAD

**Issue by lassoan** [Slicer/Slicer#5088]
This error is logged when using Undo function after Grow from seeds is initialized:

```
Python 3.6.7 (default, Aug  2 2020, 23:22:19) [MSC v.1924 64 bit (AMD64)] on win32
>>> 
Loading Slicer RC file [C:/Users/andra/.slicerrc.py]
self.extentGrowthRatio = 0.1
masterImageExtent = (0, 255, 0, 255, 0, 129)
labelsEffectiveExtent = (104, 165, 91, 158, 49, 93)
labelsExpandedExtent = [98, 171, 85, 164, 45, 97]
Grow-cut operation on volume of 74x80x53 voxels was completed in 0.3 seconds.
Traceback (most recent call last):
  File "C:\Users\andra\AppData\Local\NA-MIC\Slicer 4.11.0-2020-08-02\lib\Slicer-4.11\qt-scripted-modules\SegmentEditorEffects\AbstractScriptedSegmentEditorAutoCompleteEffect.py", line 166, in onSegmentationModified
    self.selectedSegmentModifiedTimes[segmentID] = segmentLabelmap.GetMTime()
AttributeError: 'NoneType' object has no attribute 'GetMTime'
Traceback (most recent call last):
  File "C:\Users\andra\AppData\Local\NA-MIC\Slicer 4.11.0-2020-08-02\lib\Slicer-4.11\qt-scripted-modules\SegmentEditorEffects\AbstractScriptedSegmentEditorAutoCompleteEffect.py", line 163, in onSegmentationModified
    and segmentLabelmap.GetMTime() == self.selectedSegmentModifiedTimes[segmentID]:
AttributeError: 'NoneType' object has no attribute 'GetMTime'
Grow-cut operation on volume of 74x80x53 voxels was completed in 0.0 seconds.
self.extentGrowthRatio = 0.5
masterImageExtent = (0, 255, 0, 255, 0, 129)
labelsEffectiveExtent = (104, 165, 127, 127, 49, 100)
labelsExpandedExtent = [74, 195, 124, 130, 24, 125]
Grow-cut operation on volume of 122x7x102 voxels was completed in 0.1 seconds.
Traceback (most recent call last):
  File "C:\Users\andra\AppData\Local\NA-MIC\Slicer 4.11.0-2020-08-02\lib\Slicer-4.11\qt-scripted-modules\SegmentEditorEffects\AbstractScriptedSegmentEditorAutoCompleteEffect.py", line 166, in onSegmentationModified
    self.selectedSegmentModifiedTimes[segmentID] = segmentLabelmap.GetMTime()
AttributeError: 'NoneType' object has no attribute 'GetMTime'
Traceback (most recent call last):
  File "C:\Users\andra\AppData\Local\NA-MIC\Slicer 4.11.0-2020-08-02\lib\Slicer-4.11\qt-scripted-modules\SegmentEditorEffects\AbstractScriptedSegmentEditorAutoCompleteEffect.py", line 163, in onSegmentationModified
    and segmentLabelmap.GetMTime() == self.selectedSegmentModifiedTimes[segmentID]:
AttributeError: 'NoneType' object has no attribute 'GetMTime'
Grow-cut operation on volume of 122x7x102 voxels was completed in 0.0 seconds.
```

## Steps to reproduce

- Load MRHead
- Paint a few strokes with two segments
- Select Grow from seeds
- Click initialize
- Click Undo -> error is logged

## Expected behavior

No error should occur.

## Environment
- Slicer version: Slicer-4.11.0-2020-08-02
- Operating system: Windows 10

**Comment by Sunderlandkyl** [comment]
This error started occurring since the merged labelmap rework due to additional SegmentModified/MasterRepresentationModified events that are invoked during segment copy (as a result of undo/redo).

I'm still looking at reducing the number of invoked events, however the immediate error should be fixed by #5089.

**Comment by lassoan** [comment]
Thanks a lot for the quick fix!
